dynamiqs/utils/vectorization.py

```diff
@@ -223,6 +223,10 @@ def slindbladian(H: ArrayLike, jump_ops: ArrayLike) -> Array:
     Note:
         This superoperator is also sometimes called *Liouvillian*.
 
+    Note:
+        See also [`dq.lindbladian`][dynamiqs.utils.quantum_utils.general.lindbladian] to
+        access the Lindbladian only through matrix-vector product.
```

Comment:
Maybe with the following formatting, which is indeed closer to JAX? We don't have a specific admonition for this, but maybe we could add one.
```suggestion
    Note: See also
        - [`dq.lindbladian`][dynamiqs.utils.quantum_utils.general.lindbladian]:
        access the Lindbladian only through matrix-vector product.
```

Reply:
I think it would be useful to add one indeed. Those links are really useful for newcomers in those big libraries  dynamiqs belongs to. 
If you do that I'll wait for the formatting before changing my commit

Reply:
Did that in https://github.com/dynamiqs/dynamiqs/pull/787 :)